client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx

```diff
@@ -29,14 +32,13 @@ const AddGoogleAppsCard = React.createClass( {
 	mixins: [ analyticsMixin( 'domainManagement', 'email' ) ],
 
 	render() {
-		const gapps = this.props.products.gapps,
+		const prices = get( this.props, 'products.gapps.prices', [] ),
```

Comment:
Update: D6292-code will fix the problem of prices not being available in all currencies, but support for the new currencies was added in #15492 which was merged after this PR was created, so you will still get `null` for `monthlyPrice` and `annualPrice` in this branch.